purchases/src/main/kotlin/com/revenuecat/purchases/common/OfferingParser.kt

```diff
@@ -6,12 +6,20 @@ import com.revenuecat.purchases.Offerings
 import com.revenuecat.purchases.Package
 import com.revenuecat.purchases.PackageType
 import com.revenuecat.purchases.models.StoreProduct
+import com.revenuecat.purchases.paywalls.PaywallData
 import com.revenuecat.purchases.strings.OfferingStrings
 import com.revenuecat.purchases.utils.toMap
+import kotlinx.serialization.decodeFromString
+import kotlinx.serialization.json.Json
 import org.json.JSONObject
 
 internal abstract class OfferingParser {
 
+    // TODO-PAYWALLS: uncomment after testing
+    private val json = Json {
+//        ignoreUnknownKeys = true
+    }
```

Comment:
I will refactor this and add it as a dependency in a followup PR. Also I think until we make sure all keys are there, we should not ignore unknown keys